### DISCUSSION TITLE
目录 图录 表录问题

### DISCUSSION THREAD

**Discussion by chenaamao** [sjtug/SJTUThesis#678] [提问] [answered]
生成的目录中章节与章节之间有空行，怎么去掉这个空行呢？ 图录表录均有这个问题。
![image](https://user-images.githubusercontent.com/57710595/120737255-51fb9680-c520-11eb-8809-49f8d974d81d.png)

**Comment by skyzh** [comment, marked as answer]
可以参考 https://tex.stackexchange.com/questions/89103/remove-vertical-space-between-two-chapters-in-table-of-contents-in-latex

**Reply by chenaamao** [reply]
这个网页提到的方法能够解决目录的问题，但图录和表录不变
我找到的解决方案
将main.tex中的
\listoffigures
\listoftables
改为如下代码：
<img width="208" alt="2021-06-04 (2)" src="https://user-images.githubusercontent.com/57710595/120752158-cbed4900-c53b-11eb-8170-7681a77d1f85.png">
